lib/openstax/aws/stack.rb

```diff
@@ -359,20 +330,20 @@ def wait_for_deletion
 
     def resource(logical_id)
       stack_resource = aws_stack.resource(logical_id)
+      resource_factory = OpenStax::Aws::ResourceFactory.new region: region
+      resource_factory.from_stack_resource_or_summary! stack_resource
```

Comment:
This is probably a breaking change? Returning our object instead of the AWS object.  Or does it work because of the delegate_missing stuff?

Reply:
I noticed now that we already returned our own resource for everything except the autoscaling group... hmm in theory it should work unless someone was switching based on the returned class...